paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/HeaderTextFactory.kt

```diff
@@ -86,7 +86,7 @@ internal class HeaderTextFactory(
 
     private fun getHeaderTextForManageScreen(isEditing: Boolean): Int {
         return if (isEditing) {
-            R.string.stripe_paymentsheet_manage_your_payment_methods
```

Comment:
yeah, we use it in customer sheet. I asked about updating the string in customer sheet, but it looks like we haven't thought about whether we should do that or not, since it's out of scope for vertical mode: https://stripe.slack.com/archives/C06JTLU0K8E/p1718141371951649